requirements.txt

```diff
@@ -27,8 +26,6 @@ pexpect==4.1.0
 pathlib2==2.1.0
 pickleshare==0.7.2
 simplegeneric==0.8.1
-path.py==8.2.1
-backports.shutil_get_terminal_size==1.0.0
```

Comment:
it looks like pip recommends using the original name: https://github.com/pypa/pip/issues/3757 I will put them back in.